Tests/SwiftUIKitTests/Core/BasicSwiftUIKitTests.swift

```diff
@@ -31,9 +31,37 @@ final class BasicSwiftUIKitTests: XCTestCase {
             viewToEmbed
         }
         
+        let leadingConstraint = view.constraints.first { (constraint) -> Bool in
+            constraint.firstAnchor == view.leadingAnchor || constraint.secondAnchor == view.leadingAnchor
+        }
+        
+        let bottomConstraint = view.constraints.first { (constraint) -> Bool in
+            constraint.firstAnchor == view.bottomAnchor || constraint.secondAnchor == view.bottomAnchor
```

Comment:
i wonder if this could be pulled out to a utility function

Reply:
You think instead of 
```
view.constraints.first
```

It could be
```
view.bottomConstraints.first
```


Then I could have
```
view.topConstraints
view.bottomConstraints
view.leadingConstraints
view.trailingConstraints
```

Reply:
that seem's slightly cleaner, i just saw this logic repeated several times :)